src/components/Missions.js

```diff
@@ -1,25 +1,58 @@
 import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { getMissions } from '../redux/missionsSlice';
+import { getMissions, joinMission, leaveMission } from '../redux/missions/missionsSlice';
+import '../styles/Missions.css';
```

Comment:
- [ ] You have a typo here that prevents the app from running, kindly make sure the `missions.css` is capitalized or change `Mission.css` to `mission.css`